backend/TraderX/src/main/java/cmpe451/group6/rest/equipment/service/ScheduledUpdateService.java

```diff
@@ -19,19 +19,22 @@
 
     public final long MIN_IN_MS = 1000 * 60;
 
-    @Scheduled(cron = "0 15 * * * *")
+    // Disabled for the ease of development. Uncomment before deploy
+    //@Scheduled(cron = "0 15 * * * *")
```

Comment:
Also these parts, should we keep it commented-out when merging to backend-dev?

Reply:
See reply: https://github.com/bounswe/bounswe2019group6/pull/178#discussion_r346690029